bin/topological-inventory-inventory-upload

```diff
@@ -22,5 +22,5 @@ end
 
 args = parse_args
 
-topological_inventory_sync = TopologicalInventory::Sync::InventoryUploadWorker.new(args[:queue_host], args[:queue_port])
+topological_inventory_sync = TopologicalInventory::Sync::InventoryUpload::ProcessorWorker.new(args[:queue_host], args[:queue_port])
```

Comment:
@agrare please don't make changes in upload worker in this PR, it'll cause merge conflicts

Reply:
That's life sometimes :)

Reply:
well, ok, but this PR doesn't do anything with this class except rename :)

Reply:
It moves it into a module with the other inventory_upload code